src/utilities/formInputConfigs.js

```diff
@@ -221,7 +221,7 @@ const FORM_INPUT_CONFIGS = seedObject => ({
     type: FORM_INPUT_TYPES.TEXT,
     initialValue: '',
     key: 'policyNumberPerson',
-    validator: null,
+    validator: input => input.length > 0 && input.length < 50,
```

Comment:
Yes, I know. we've been here before. You may safely ignore this comment.
In desktop, only the family number is mandatory now.